python/py-pymc3/Portfile

```diff
@@ -4,33 +4,35 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        pymc-devs pymc3 aca6086e385a5125fdc09f2df24c09baabe23436
+github.setup        pymc-devs pymc3 3.0 v
+# github.tarball_from releases
```

Comment:
Leaving this commented out is not helpful, as the intention is not clear. If it is not needed, just remove it or add a comment with an explanation.